dockeradapter/versions/versions.go

```diff
@@ -0,0 +1,10 @@
+package versions
```

Comment:
Odd question, but did this not make it in https://github.com/puppetlabs/lumogon/pull/57/files#diff-c66508b17e894eae7c5df967108d55f2R1? Needs a merge from master?

Reply:
don't quite follow, whats missing here?

Reply:
I've rebased post-#57 merge hopefully that sorts it?